src/scripts/Package-Toolkit.ps1

```diff
@@ -93,50 +97,44 @@ $templates = Get-ChildItem $relDir -Directory `
         }
     }
 
-    Write-Verbose "Updating $($path.Name) deployment file in docs..."
+    Write-Verbose "Updating $($path.Name) deployment files in docs..."
 
-    $packageManifestPath = "$path/package-manifest.json"
-    if (Test-Path $packageManifestPath)
+    function Copy-DeploymentFiles($suffix)
     {
-        $packageManifest = Get-Content $packageManifestPath -Raw | ConvertFrom-Json
-        $docsDeployDir = $deployDir
-        if ($packageManifest.deploySubDir)
-        {
-            $docsDeployDir = "$deployDir/$($packageManifest.deploySubDir)"
-            & "$PSScriptRoot/New-Directory" $docsDeployDir
-        }
-        if ($packageManifest.deploySubDirVersioned)
-        {
-            $docsDeployDirVersioned = "$deployDir/$($packageManifest.deploySubDirVersioned.Replace('{version}', $version))"
-            & "$PSScriptRoot/New-Directory" $docsDeployDirVersioned
-        }
-        foreach ($file in $packageManifest.deployFiles)
+        $packageManifestPath = "$path/package-manifest.json"
+        if (Test-Path $packageManifestPath)
         {
-            Copy-Item "$path/$($file.source)" "$docsDeployDir/$($file.destination.Replace('{version}', $version))"
-            if ($packageManifest.deploySubDirVersioned)
-            {
-                Copy-Item "$path/$($file.source)" "$docsDeployDirVersioned/$($file.destination.Replace('{version}', $version))"
+            # Read files/directories from package-manifest.json
+            $packageManifest = Get-Content $packageManifestPath -Raw | ConvertFrom-Json
+
+            # Create release directory
+            $targetDir = "$deployDir/$($path.Name)/$suffix"
+            & "$PSScriptRoot/New-Directory" $targetDir
+            
+            # Copy files and directories
+            $packageManifest.deployment.Files | ForEach-Object { Copy-Item "$path/$($_.source)" "$targetDir/$($_.destination)" }
+            $packageManifest.deployment.Directories | ForEach-Object {
+                & "$PSScriptRoot/New-Directory" "$targetDir/$($directory.destination)"
+                Get-ChildItem "$path/$($directory.source)" | Copy-Item -Destination "$docsDeployDir/$($directory.destination)" -Recurse -Force
```

Comment:
Some lines were not completely refactored. This will break AOE deployment. Replace lines 117-118 with the following:

```suggestion
                & "$PSScriptRoot/New-Directory" "$targetDir/$($_.destination)"
                Get-ChildItem "$path/$($_.source)" | Copy-Item -Destination "$targetDir/$($_.destination)" -Recurse -Force
```

Reply:
Good catch!